experiment/evaluator.py

```diff
@@ -217,7 +291,34 @@ def _parse_libfuzzer_logs(self, log_handle) -> tuple[int, int, bool]:
         crashes = True
         continue
 
-    return cov_pcs, total_pcs, crashes
+    initcov, donecov, lastround = self._parse_fuzz_cov_info_from_libfuzzer_logs(
+        lines)
+
+    # NOTE: false positive crash will not be counted as crash
```

Comment:
nit: Capitalise the first char after `:`, e.g., `# NOTE: False ...`.
Same below.